docs/semgrep-sc/sc-getting-started.md

```diff
@@ -21,135 +28,70 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Scanning open source dependencies
+# Scanning open source dependencies 
 
-<SscIntro />
+Semgrep Supply chain scans can be triggered from the Semgrep App interface.
 
-<AdmonitionSscLicense />
-
-![Semgrep Supply chain overview page](/img/sc-overview.png)
-_Figure 1_. Semgrep Supply Chain overview page.
+:::note
+Semgrep Supply Chain **supports monorepositories (monorepos)** by treating each subdirectory as its own repository. Findings are grouped under these repositories based on the lockfile or manifest file present in the subdirectory.
+:::
 
-Semgrep Supply Chain parses lockfiles for a list of dependencies, then scans your codebase using **rules **written with Semgrep's pattern syntax. Supply Chain rules specify the following:
+## Scanning with Semgrep Supply Chain through Semgrep App
 
-* The range of versions that contain the dependency's vulnerability.
-* A pattern for vulnerable code, such as passing in unsanitized data.
-* The severity of the vulnerability.
+:::tip
+This is the preferred method to enable and run Semgrep Supply Chain.
+:::
 
-Semgrep Supply Chain generates a **finding **when it detects a match. If the dependency's version is within the range and finds the matching code within your codebase, the finding is **reachable**.
+Perform the following steps to create an account in Semgrep App and add (onboard) a repository for scanning. By adding a repository to Semgrep App, you are able to set up Semgrep Supply Chain scans and receive findings for vulnerabilities which you can triage and remediate.
 
-A finding is **unreachable** if the dependency contains a known vulnerability, but the vulnerable matching code is not used in your codebase.
+Read [Step 3](#step-3-enabling-semgrep-supply-chain-scans-from-within-semgrep-app) if you have already created an account in Semgrep App and added a repository.
 
-Within Semgrep App, specific findings of a dependency and code match are called **usages**. Usages are grouped by their **vulnerability**. Vulnerabilities in Semgrep Supply Chain typically have a CVE number corresponding to the record in the [CVE Program](https://www.cve.org/About/Overview).
+### Step 1: Creating an account in Semgrep App
```

Comment:
Remove Step 1:, Step 2:, Step 3, Step 4 from the headers - when linking to specific sections (`Read [Step 3](#step-3-enabling-semgrep-supply-chain-scans-from-within-semgrep-app)`) reference the section header (`Follow instructions in [Enabling Semgrep Supply Chain scans from within Semgrep App]).

Every list item in a procedure is already a step. Creating "groups of steps" called step 1, step 2 is not a good practice in documentation.

Reply:
Sure, I'll remove it, but I also have to point out that a lot of popular docs use "Step 1" in their headers, too: [https://docs.github.com/en/enterprise-server@3.5/developers/apps/guides/using-the-github-api-in-your-app](https://docs.github.com/en/enterprise-server@3.5/developers/apps/guides/using-the-github-api-in-your-app)

Reply:
Maybe take this suggestion as optional also. I was just taught to avoid it, but we don't have to be that strict (sorry for uncompromising tone in the first suggestion). @s-santillan